### PR TITLE
Flask fixes.

### PR DESCRIPTION
**Pull request by oliverchang**
Add missing return value for /commit-range/load handler.

Also call make_response in the various wrappers in handler_flask to
ensure that we have a Response object.

### CODE REVIEW

File: src/appengine/libs/handler_flask.py [hunk 1]
```diff
@@ -21,12 +21,14 @@
 import re
 import requests
 
+from flask import g
```
Comment (inferno-chromium) [inline src/appengine/libs/handler_flask.py:24]: remove ?

Comment (oliverchang) [reply src/appengine/libs/handler_flask.py:24]: This is needed for per-request caching: https://flask.palletsprojects.com/en/1.1.x/api/#flask.g

Comment (inferno-chromium) [reply src/appengine/libs/handler_flask.py:24]: :)

**Pull request merged**